src/internal/kopia/snapshot_manager.go

```diff
@@ -202,7 +202,7 @@ func fetchPrevManifests(
 	return manifestsSinceLastComplete(mans), nil
 }
 
-// fetchPrevSnapshotManifests returns a set of manifests for complete and maybe
+// FetchPrevSnapshotManifests returns a set of manifests for complete and maybe
```

Comment:
still lost in rebase?
```suggestion
// fetchPrevSnapshotManifests returns a set of manifests for complete and maybe
```

Reply:
bugger.  Yes, thank you.